pkg/envoy/accesslog_server.go

```diff
@@ -39,6 +39,12 @@ func StartAccessLogServer(stateDir string, xdsServer *XDSServer) {
 	}
 	accessLogListener.SetUnlinkOnClose(true)
 
+	// Make the socket accessible by non-root Envoy proxies, e.g. running in
+	// sidecar containers.
+	if err = os.Chmod(accessLogPath, 0777); err != nil {
```

Comment:
Does this mean that the app itself could log record? 😨 Can we restrict it to a group?

Reply:
OK